pkgs/tools/text/mdbook/default.nix

```
@@ -13,8 +13,18 @@ rustPlatform.buildRustPackage rec {
 
   cargoHash = "sha256-D0XhrweO0A1+81Je4JZ0lmnbIHstNvefpmogCyB4FEE=";
 
+  nativeBuildInputs = [ installShellFiles ];
+
   buildInputs = lib.optionals stdenv.isDarwin [ CoreServices ];
 
+  postInstall = let emulator = stdenv.hostPlatform.emulator buildPackages; in
```

```suggestion
  postInstall = lib.optionalString (stdenv.buildPlatform.canExecute stdenv.hostPlatform) ''
```
other packages are just using this because an emulator can be quite a heavy lifter to just generate some script files.